js/clinicalTimeline.js

```diff
@@ -261,6 +283,139 @@ window.clinicalTimeline = (function(){
+    var minDayTick = tickTemp[0];
+    var maxDayTick =  tickTemp[tickTemp.length-1];
+    var zoomedWidth = chart.width();
+
+    var overview = d3.select(divId).append("svg")
+      .attr("height", 75)
+      .attr("width", width)
+      .attr("class", "overview");
+
+    var xScaleOverview = d3.time.scale()
+      .domain([minDayTick, maxDayTick])
+      .range([0 , width]);
+
+    var xScaleOverviewZoomed = d3.time.scale()
+      .domain([0, -zoomedWidth])
+      .range([0 , width]);
+
+    var xScaleRectangle = d3.time.scale()
+      .domain([0, width])
+      .range([0 , -zoomedWidth]);
+
+    var overviewAxis = d3.svg.axis().scale(xScaleOverview).orient("bottom")
+      .tickFormat(function(d) { 
+        return formatTime(daysToTimeObject(d.valueOf()), "months");
+      })
+      .ticks(tickTemp.length)
+      .tickSize(3)
+      .tickPadding(4);
+
+     var overviewAxisDuplicate = d3.svg.axis().scale(xScaleOverview).orient("top")
+      .tickFormat(function(d) { 
+        return formatTime(daysToTimeObject(d.valueOf()), "months");
+      })
+      .ticks(tickTemp.length)
+      .tickSize(3)
+      .tickPadding(0);
+
+    var rectangleOverviewWidth = width/zoomFactor;
+
+    overview.append("g")
+      .attr("class", "x axis overviewaxes")
+      .attr("transform", "translate(0,25)")
+      .call(overviewAxis);
+
+    overview.append("g")
+      .attr("class", "x axis overviewaxes duplicate")
+      .attr("transform", "translate(0,44)")
+      .call(overviewAxisDuplicate);
+
+    var dragChart = d3.behavior.drag()
+      .on("drag", function(d,i) {
+        if(chart.scrolledX()){
+          var zoomedBeginTick = xScaleOverviewZoomed(chart.scrolledX()); 
+          d3.select(".pos_rectangle").attr("x", zoomedBeginTick);
+          overviewX = zoomedBeginTick;
+        }
+      });
+
+    var dragRectangle = d3.behavior.drag()
+      .on("drag", function(d,i) {
+        var x  = parseInt(d3.select(".pos_rectangle").attr("x"))+d3.event.dx;
+        if(x > 0 && x < width - rectangleOverviewWidth){
+          d3.select(divId+" svg g").attr("transform","translate("+xScaleRectangle(x)+",0)");
+          d3.select(".pos_rectangle").attr("x", x); 
+          overviewX = x;
+        }
+      });
+
+    d3.select(".overview").append("rect")
+      .attr("height", 3)
+      .attr("width", width)
+      .attr("x", 0)
+      .attr("y", 22)
+      .attr("fill", "#ccc");
+
+    d3.select(".overview").append("rect")
+      .attr("height", 3)
+      .attr("width", width)
+      .attr("x", 0)
+      .attr("y", 44)
+      .attr("fill", "#ccc");
+
+    d3.select(".overview").append("rect")
+      .attr("height", 18)
+      .attr("width", 3)
+      .attr("x", 0)
+      .attr("y", 26)
+      .attr("fill", "#ccc");
+
+    d3.select(".overview").append("rect")
+      .attr("height", 18)
+      .attr("width", 3)
+      .attr("x", width-3)
+      .attr("y", 26)
+      .attr("fill", "#ccc");
+
+
+    if(advancedView){
```

Comment:
`advancedView` is a separate feature, so it's better to separate it from this PR. I think this part should also be outside of `handleOverviewAxis` since you don't want to draw the overviewAxis at all if `advancedView` is false.